app/concepts/reservations/use_cases/create.rb

```diff
@@ -8,7 +8,7 @@ def initialize(repository: Reservations::Repository.new)
       end
 
       def call(params:)
-        repository.create(params.merge(reservation_status_id: 1))
+        repository.create(params.merge(reservation_status: ReservationStatus.booked))
```

Comment:
Could you explain to me what does this line of code do?

Reply:
I used a static statuses in the db and here I set the default status for a new reservation (which is always "booked" status, in my app). You can see what the "booked" method does in the ReservationStatus class :)